packages/editor/src/components/post-actions/actions.js

```diff
@@ -39,21 +38,31 @@ import { getItemTitle } from '../../dataviews/actions/utils';
 const { PATTERN_TYPES, CreatePatternModalContents, useDuplicatePatternProps } =
 	unlock( patternsPrivateApis );
 
-// TODO: this should be shared with other components (page-pages).
+// TODO: this should be shared with other components (see post-fields in edit-site).
```

Comment:
https://github.com/WordPress/gutenberg/pull/63849 addresses this but it's still unclear how to do it.